src/main/webapp/script.js

```diff
@@ -75,26 +77,29 @@ function searchLocation() {
     htmlString += "<h3>Wagga Wagga</h3>\n";
     htmlString += "<p>Current AQ score: 40 (Pretty good)</p>\n";
     locationInfo.innerHTML = htmlString;
-
-    zoomedIn = true;
-
-    routeVisible = false; //no image available for zoomed in and routes visible
-    document.getElementById("close-directions").style.visibility = "hidden";
-    document.getElementById("route-info").style.visibility = "hidden";
-
-    changeBackgroundImage(getImagePath());
 }
 
-function closeSidebar() {
-    document.getElementById("close-sidebar").style.display = "none";
-    document.getElementById("open-sidebar").style.display = "inline-block";
+function showRoutes() {
+    console.log("Directions not currently supported");
+}
 
-    document.getElementById("sidebar").style.visibility = "hidden";
+function hideRoutes() {
+    console.log("Directions not currently supported");
 }
 
-function openSidebar() {
-    document.getElementById("close-sidebar").style.display = "inline-block";
-    document.getElementById("open-sidebar").style.display = "none";
+function toggleSidebar() {
+    const toggleButton = document.getElementById("toggle-sidebar");
 
-    document.getElementById("sidebar").style.visibility = "visible";
+    const sidebar = document.querySelector("#sidebar");
+    const sidebarVisibility = window.getComputedStyle(sidebar).getPropertyValue("visibility");
+    
+    if (sidebarVisibility === "visible") {
+        sidebar.style.visibility = "hidden";
+        toggleButton.innerHTML = "open sidebar";
```

Comment:
If the label was "toggle sidebar" you wouldn't need to keep changing it.

Reply:
Good point, but since I'm planning on changing it to arrow icons later on, I'll leave it be